### DISCUSSION TITLE
[Feature Request] Autocomplete possibility

### DISCUSSION THREAD

**Discussion by ghost** [heroui-inc/heroui#1398] [Feature requests]
### Is your feature request related to a problem? Please describe.

I'm looking for a component / example how to implement a autocomplete feature in NextUI. Something like the google search bar or a street finder.

### Describe the solution you'd like

The user has a text input field and as soon as they type, the app can send a request to the backend and render the recieved values in some sort of drop down field.

### Describe alternatives you've considered

MUI has such a mechanism for example: https://mui.com/joy-ui/react-autocomplete/
I'm also thinking about implementing this with a regular NextUI dropdown component but this seems hacky

**Comment by jguddas** [comment]
https://react-spectrum.adobe.com/react-aria/useComboBox.html#allow-custom-values

**Comment by mauriciabad** [comment]
Another implementation example, and uses Tailwind:
https://headlessui.com/react/combobox

In my opinion, this would be a very useful component or variant.
